manifests/release/vmpreset-windows.yaml.in

```diff
@@ -0,0 +1,33 @@
+apiVersion: kubevirt.io/v1alpha1
```

Comment:
Do we want to add it to kubevirt.yaml.in, or ship presets (maybe also presets) in a separate yaml?

Reply:
Not sure. In my opinion maybe worth adding a folder just for presets.

Do we want to ship presets with matching VirtualMachine definitions ? 
We have the /cluster/windows-demo.yaml  as well, do we want to create another demo file just for the preset (remove the content the preset already have) ?

Reply:
What about `kubevirt.yaml` for the "operator" and `demo-content.yaml` for presets and some other (once landed) `OfflineVirtualMachine`s which can easily be started.

Reply:
+1 I'm for Kubevirt components in one manifest. Anything that kubevirt consumes (like presets) in other manifests.